release/release_tests.yaml

```diff
@@ -2629,7 +2629,16 @@
 
   frequency: weekly
   team: serve
+  python: "3.8"
   cluster:
+    byod: 
+      runtime_env:
+        - RLLIB_TEST_NO_JAX_IMPORT=1
+      pre_run_cmds:
+        - rm -rf /tmp/wrk 
+        - git clone https://github.com/wg/wrk.git /tmp/wrk 
```

Comment:
shall we pin to a tag of wg/wrk here? like https://github.com/wg/wrk/releases/tag/4.2.0

the repo has no updates in 2 years though.

since this is repeated in a lot of tests, should we make it a bash script (like `serve_test_init.sh`), and run that?

Reply:
i like that idea, but is there a generic way to upload serve_test_init.sh as part of an anyscale job?

Reply:
@sven1977 , @avnishn , @ArturNiederfahrenhorst , do you have context on this wrk tool; can it be installed as a ubuntu debian package, instead of via github? thankks

Reply:
I've never heard of it and looking through our codebase it seems like only `ray/serve/benchmarks/baselines/Makefile` uses it?

Reply:
I can install it with brew.

Reply:
Coo cool, I'll switch to debian or something and see if the tests still pass

Reply:
Awesome! 👍